internal/attestors/github_pull_request/github_pull_request.go

```diff
@@ -28,12 +29,18 @@ const (
 	GitHubReviewApprovedState         = "APPROVED"
 )
 
-type GitHubPullRequestAttestor struct {
+type ctxKey struct{}
+
+var (
+	GitRepositoryOverrideCtxKey = ctxKey{}
```

Comment:
I'm using this to control the repository in-use for different tests. Normally, this would be the working directory, but that would make it difficult to run tests in parallel. 

Instead, the tests make an in-memory `go-git` repository with a single commit, and that's what the attestor sees.